workos/types/sso/profile.py

```diff
@@ -23,7 +23,8 @@ class Profile(WorkOSModel):
     idp_id: str
     role: Optional[ProfileRole] = None
     groups: Optional[Sequence[str]] = None
-    raw_attributes: Mapping[str, Any]
+    custom_attributes: Optional[Mapping[str, Any]] = None
```

Comment:
I'd like to add a generic type for custom_attributes that developers can override, but ran into some mypy limitations with default values for generic types. Keeping this as-is for now.